packages/web3-eth-contract/test/integration/contract_methods.test.ts

```diff
@@ -148,23 +148,35 @@ describe('contract', () => {
 			});
 
 			it('should returns errors on reverts', async () => {
-				await expect(
-					contractDeployed.methods.reverts().send(sendOptions),
-				).rejects.toMatchObject({
-					name: 'TransactionRevertedWithoutReasonError',
-					receipt: {
-						cumulativeGasUsed: BigInt(21543),
-						from: acc.address,
-						gasUsed: BigInt(21543),
-						logs: [],
-						logsBloom:
-							'0x00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000',
-						status: BigInt(0),
-						to: contractDeployed.options.address?.toLowerCase(),
-						transactionIndex: BigInt(0),
-						type: BigInt(2),
-					},
-				});
+				// hardhat does not support eip-838
+				if (getSystemTestBackend() === 'hardhat') {
```

Comment:
I'll suggest to create one const key and use it in tests instead of hardhat string in multiple places,